pages/linux/swapoff.md

```diff
@@ -0,0 +1,19 @@
+# swapoff
+
+> Disables device or file for swapping.
+
+- Disable a given swap partition:
+
+`swapoff {{/dev/sdb7}}`
```

Comment:
Could you change this to `path/to/partition`?

Reply:
Thanks for suggesting edit, but `{{/dev/sdb7}}` is more consistent with other pages such as `mkswap` and `swapon`.

Reply:
Seems fine to me then. :+1: